MOxUnit/assertGreaterThan.m

```diff
@@ -0,0 +1,54 @@
+function assertGreaterThan(a, b, message)
+% assert that a is greater than b
+%
+% assertGreaterThan(a,b,[msg])
+%
+% Inputs:
+%   a           first input  } of any
+%   b           second input } type
+%   msg             optional custom message
+%
+% Raises:
+%   'assertGreaterThan:classNotEqual'        a and b are of different class
+%   'assertGreaterThan:notGreaterThan'       values in a must be greater
+%                                            than values in b
+%
+% Examples:
+%   assertGreaterThan(2,1);
+%   %|| % passes without output
+%
+%   assertGreaterThan(1,2);
+%   %|| error('first input argument in not larger than the second');
+%
+%   assertGreaterThan([2 3],[0;1]);
+%   %|| % passes without output
+
+    if ~isequal(class(a), class(b))
+        whatswrong='inputs are not of the same class';
+        error_id='assertGreaterThan:classNotEqual';
+
+    elseif length(a) == 1 && length(b) == 1 && a <= b
```

Comment:
NaNs behave special so this doesn't work for them I think; instead of ``a<=b`` use ``~(a > b)``